data/darktableconfig.xml.in

```diff
@@ -2071,11 +2078,74 @@
   </dtconfig>
   <dtconfig>
     <name>plugins/imageio/format/png/compression</name>
-    <type>int</type>
+    <type min="0" max="9">int</type>
     <default>5</default>
     <shortdescription/>
     <longdescription/>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/imageio/format/webp/comp_type</name>
+    <type min="0" max="1">int</type>
+    <default>0</default>
+    <shortdescription/>
+    <longdescription/>
+  </dtconfig>
+  <dtconfig>
+    <name>plugins/imageio/format/webp/quality</name>
+    <type min="5" max="100">int</type>
+    <default>95</default>
+    <shortdescription/>
+    <longdescription/>
+  </dtconfig>
+  <dtconfig>
+    <name>plugins/imageio/format/webp/hint</name>
+    <type>int</type>
+    <default>0</default>
+    <shortdescription/>
+    <longdescription/>
+  </dtconfig>
+  <dtconfig>
+    <name>plugins/imageio/format/avif/bit_depth</name>
```

Comment:
maybe an enum with 8 / 10 / 12 as possible values ?

Reply:
actually I'd rather follow convention of other formats, that is rename `bit_depth` to `bpp` and have it still int, so that it's in line with other formats... And other formats still are limited, eg xmp likes only 8/16/32 and jpeg just 8 :)

of course for renaming I'd like to have a greenlight from @cryptomilk for that

Reply:
using an enum ensure that the conf circuitry will be able to restore a proper supported values in case of corruption. a bit_depth of 9 is not supported, would be better to detect this. If some format support other bit_depth I would rather have another enum with the proper list of values.

Reply:
so... maybe change all other bpp & this one to enum to exactly what format accepts?

Reply:
Yes, that's what I'm thinking.

Reply:
Feel free to rename it to have it consistent. However I find bpp not intuitive. Use a search engine and search for "bpp" and then for "bit depth". The second gives you proper results ;-)

I agree bit_depth should be an enum, however as the config only supported int I just used it everywhere.

Reply:
This is now done in recent commit